src/NfMaxOrd/ResidueRing.jl

```diff
@@ -171,6 +173,24 @@ end
 
 *(x::NfMaxOrdQuoRingElem, y::fmpz) = y*x
 
+function ^(a::NfMaxOrdQuoRingElem, f::fmpz)
+  f==0 && return one(parent(a))
+  f==1 && return a
+  if f<0
+    f=-f
+    a = inv(a)
+  end
+  b = a^(div(f, 2))
+  b *= b
+  if isodd(f)
+    b *= a
+  end
+  return b
+end
+
+^(a::NfMaxOrdQuoRingElem, f::Integer) = a^fmpz(f)
```

Comment:
Hätte ich jetzt auch gedacht. Leider sagt Julia 
`WARNING: New definition 
    ^(Hecke.NfMaxOrdQuoRingElem, #T<:Union{Integer, Nemo.fmpz}) at /home/alex/.julia/v0.4/Hecke/src/NfMaxOrd/ResidueRing.jl:177
is ambiguous with: 
    ^(Any, Integer) at intfuncs.jl:108.
To fix, define 
    ^(Hecke.NfMaxOrdQuoRingElem, _<:Integer)
before the new definition.`